Packs/FeedGithub/Integrations/FeedGithub/FeedGitHub.yml

```diff
@@ -0,0 +1,172 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: Github Feed
+  version: -1
+configuration:
+- defaultvalue: 'true'
+  display: Fetch indicators
+  name: feed
+  required: false
+  type: 8
+- defaultvalue: https://api.github.com
+  display: Server URL
+  name: url
+  type: 0
+  additionalinfo: The REST API URL
+  section: Connect
+  required: false
+- defaultvalue: ""
+  display: Owner
+  name: owner
+  type: 0
+  additionalinfo: Username of the repository owner
+  section: Connect
+  required: true
+- defaultvalue: ""
+  display: Repository / Path to fetch
+  name: repo
+  type: 0
+  additionalinfo: The repository name
+  section: Connect
+  required: true
+- defaultvalue: ""
+  display: Feed type
+  name: feedType
+  options:
+  - YARA
+  - STIX
+  - IOCs
```

Comment:
check with @amit-dror1993 if we want the third option to be called "IOCS". Maybe REGEX_PARSING is a bit more clear what we want

Reply:
Ho said leave it to the IOCs